examples/nginx/docker-compose.yml

```diff
@@ -0,0 +1,64 @@
+version: "3.5"
+
+services:
+  db:
+    image: mariadb:10.5
+    container_name: anonaddy_db
+    command:
+      - "mysqld"
+      - "--character-set-server=utf8mb4"
+      - "--collation-server=utf8mb4_unicode_ci"
+    volumes:
+      - "./db:/var/lib/mysql"
+    environment:
+      - "MYSQL_ALLOW_EMPTY_PASSWORD=yes"
+      - "MYSQL_DATABASE"
+      - "MYSQL_USER"
+      - "MYSQL_PASSWORD"
+    restart: always
+
+  redis:
+    image: redis:4.0-alpine
+    container_name: anonaddy_redis
+    restart: always
+
+  anonaddy:
+    image: anonaddy/anonaddy:latest
+    container_name: anonaddy
+    depends_on:
+      - db
+      - redis
+    ports:
+      - target: 25
+        published: 25
+        protocol: tcp
+      - target: 8000
+        published: 8000
+        protocol: tcp
+      - target: 11334
+        published: 11334
+        protocol: tcp
```

Comment:
Not necessary to expose 8000 / 11334 to the host as they are already managed by nginx.